src/devtools/client/debugger/src/components/SecondaryPanes/FrameTimeline.tsx

```diff
@@ -106,15 +105,14 @@ class FrameTimeline extends Component<PropsFromRedux, FrameTimelineState> {
   };
 
   onMouseUp = (event: MouseEvent) => {
-    const { seek, clearPreviewPausedLocation } = this.props;
+    const { seek } = this.props;
 
     const progress = this.getProgress(event.clientX);
     const position = this.getPosition(progress);
     this.setState({ scrubbing: false });
 
     if (position) {
       seek(position.point, position.time, true);
-      clearPreviewPausedLocation();
```

Comment:
This fixes a jumping issue

https://www.loom.com/share/21fe570bdf2c4d48a1a6a3544d842a28